plugins/pluginmanagers/lz-n.nix

```diff
@@ -209,6 +247,15 @@ lib.nixvim.plugins.mkNeovimPlugin {
     globals.lz_n = lib.modules.mkAliasAndWrapDefsWithPriority id options.plugins.lz-n.settings;
     plugins.lz-n.luaConfig.content = mkIf (cfg.plugins != [ ]) ''
       require('lz.n').load( ${toLuaObject cfg.plugins})
+
+      -- Set up lz.n keybinds {{{
+      do
+        local __lz_n_keymaps = ${toLuaObject (map removeDeprecatedMapAttrs cfg.keymaps)}
```

Comment:
Shouldn't need `removeDeprecatedMapAttrs` in new code; it's for removing the deprecated `lua` attr.